LFPy/cell.py

```diff
@@ -1917,5 +1913,332 @@ def insert_v_ext(self, v_ext, t_ext):
             for seg in sec:
                 self.v_ext[i].play(seg._ref_e_extracellular, self.t_ext)
                 i += 1
-        
-        return 
+
+        return
+
+    def get_axial_currents(self):
```

Comment:
I think it may have been intentional but I propose that the returned arguments are shaped `(2*self.totnsegs, ...)`. Now the returned arguments are always shaped `(2*self.totnsegs-2, ...)`. I know the start-to-mid point axial current of soma section would normally be 0, as well as mid-to-end axial current of terminating segments will be zero. If number of currents are 2*self.totnsegs then one should be able to unambigously access axial currents of a segment, granted that the order of axial currents and cell segments is similar